README.md

```diff
@@ -104,18 +93,70 @@ The lighthouse report results are automatically printed to the **Deploy log** in
 2:35:17 PM: }
 ```
 
-We currently support the following settings, which are passed directly to Lighthouse:
+## Lighthouse plugin configuration options
+
+### Run Lighthouse for different site paths
```

Comment:
@jackbrewer @nasivuela You're welcome to suggest/add other examples here. Also, let me know if there's other wording that's helpful here. This is more of a bonus edit.